packages/editor/src/components/post-title/index.native.js

```diff
@@ -61,16 +58,26 @@ class PostTitle extends Component {
 		this.props.onSelect();
 	}
 
-	onPaste( { value, onChange, plainText } ) {
+	onPaste( { html, plainText } ) {
+		const { title, onInsertBlockAfter, onUpdate } = this.props;
+
 		const content = pasteHandler( {
+			HTML: html,
 			plainText,
-			mode: 'INLINE',
-			tagName: 'p',
 		} );
 
-		if ( typeof content === 'string' ) {
-			const valueToInsert = create( { html: content } );
-			onChange( insert( value, valueToInsert ) );
+		if ( typeof content !== 'string' && content.length ) {
```

Comment:
I was curious about what happens when pasting a string since it looks like that is no longer handled.
So far I have only tested Android but pasting blocks works fine but a non block paste does not:

https://user-images.githubusercontent.com/744755/146026176-61c3d0bc-3937-417c-b7e4-9bc4996cb00f.mp4

Reply:
I am on Android 12 , Pixel 5

Reply:
Thanks for catching this 🤦. I forgot the most straightforward case.